src/AmazonS3EncryptionClient.cs

```diff
@@ -111,7 +114,7 @@ protected override void CustomizeRuntimePipeline(RuntimePipeline pipeline)
             base.CustomizeRuntimePipeline(pipeline);
 
             pipeline.AddHandlerBefore<Amazon.Runtime.Internal.Marshaller>(new SetupEncryptionHandlerV1(this));
-            pipeline.AddHandlerAfter<Amazon.Runtime.Internal.Marshaller>(new UserAgentHandler("S3CryptoV1n"));
+            pipeline.AddHandlerAfter<Amazon.Runtime.Internal.Marshaller>(new UserAgentHandler($"lib/S3CryptoV1n#{_assemblyVersion}"));
```

Comment:
Should these be `ft/` instead of `lib/`? That's my read of the internal design.

Reply:
I have changed the UA string to include the `ft/` metadata prefix based on the client version being used. The updated UA would be:
* `lib/amazon-extensions-s3-encryption#2.1.1.0 ft/s3-crypto-v1`
* `lib/amazon-extensions-s3-encryption#2.1.1.0 ft/s3-crypto-v2`